### DISCUSSION TITLE
Anyone building in an AWS lambda?

### DISCUSSION THREAD

**Discussion by stefan-h-singer** [react-static/react-static#1452] [General]
Hi! We're running into size issues with our lambda and I was wondering if anyone has tried using serverless-webpack or so?

**Comment by nddery** [comment]
We're building through CodePipeline/CodeBuild without any issues. Never played with serverless-webpack or anything like that.

**Reply by stefan-h-singer** [reply]
That's great, but we've already set things up in an lambda which clearly is supported since how to do it is in the docs :)

**Comment by SleeplessByte** [comment]
We're building using GitHub Actions and GitLab CI; I can _imagine_ this being an issue on lambda tho.

How many pages / how big is the project?

**Reply by stefan-h-singer** [reply]
We use GitLab CI for certain things too, but that's more DevOps-oriented in our setup, we don't want every publish action on Contentful to trigger a deploy on our GitLab server.

The site is multilocale and it's got in total close to a hundrad pages for each locale. Only one locale is launched yet, you can have a look at https://www.lapoint.dk if you're curious :)
